_targets.R

```diff
@@ -464,7 +464,7 @@ list(
         year = year,
         sc_demographics = sc_demog_lookup %>%
           slfhelper::get_chi() %>%
-          dplyr::select(c("sending_location", "social_care_id", "chi")),
+          dplyr::select(c("sending_location", "social_care_id", "chi", "latest_flag")),
```

Comment:
I think you need to make this change on `process_lookup_sc_client` on this branch as well otherwise targets might break because its missing here

Reply:
Noticed you've done this in #979 but think it also needs to match on this one so that you dont get a conflict

Reply:
done!